docs/hashes.toml

```diff
@@ -1,72 +1,120 @@
 # Generated by: ckb cli hashes -b
+
+# Spec: ckb_dev
 [ckb_dev]
 genesis = "0x8a1737df1b0fc290c1265a92490c7d941c8dcfa1a803e711ba6d86f4290e36d7"
 cellbase = "0x57d4f9e30b71e949a126c4b9094cfecdb95b28993f638f74a9aba705c8a33389"
 
 [[ckb_dev.system_cells]]
 path = "Bundled(specs/cells/secp256k1_blake160_sighash_all)"
+tx_hash = "0x57d4f9e30b71e949a126c4b9094cfecdb95b28993f638f74a9aba705c8a33389"
 index = 1
 code_hash = "0x664b03c55410dd7d694a73779fa86edd7f069ee17fd79b0fef4c4fec0377b9a6"
 
 [[ckb_dev.system_cells]]
 path = "Bundled(specs/cells/dao)"
+tx_hash = "0x57d4f9e30b71e949a126c4b9094cfecdb95b28993f638f74a9aba705c8a33389"
 index = 2
 code_hash = "0x75ff25b427a9bee13106da176f1b5884f931432a9bf0e06fa4044fe6bc327cd5"
 
 [[ckb_dev.system_cells]]
 path = "Bundled(specs/cells/secp256k1_data)"
+tx_hash = "0x57d4f9e30b71e949a126c4b9094cfecdb95b28993f638f74a9aba705c8a33389"
 index = 3
 code_hash = "0x9799bee251b975b82c45a02154ce28cec89c5853ecc14d12b7b8cccfc19e0af4"
 
 [[ckb_dev.system_cells]]
 path = "Bundled(specs/cells/secp256k1_ripemd160_sha256_sighash_all)"
+tx_hash = "0x57d4f9e30b71e949a126c4b9094cfecdb95b28993f638f74a9aba705c8a33389"
 index = 4
 code_hash = "0x1acf88382192fbf76fc5a457a11cc393a90d84a46c5e0d7ceebfd1d0dec871e4"
 
-[ckb_staging]
-genesis = "0xa0e8f608ea2cc11ce4eb583ed7ac2228b9dfbe71eadd960085f6c27858e2643f"
-cellbase = "0x480637c97e3b43aceb4e4d60b7fad293f60b7938054d9697192524b4e158395a"
+[[ckb_dev.dep_groups]]
```

Comment:
I think it is better to add the info which system cells are included in this group.

Reply:
ok